Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rucio/rucio-clients:release-1.29.1
```

Comment:
Can you please add `README.rst` and `Makefile` as for the other example environments?